### DISCUSSION TITLE
Problem in Tutorial - Start out with EJBCA Docker container

### DISCUSSION THREAD

**Discussion by syamimirfan** [Keyfactor/ejbca-ce#451] [Q&A]
Hello, I'm having problem with starting EJBCA Docker container tutorial
![Screenshot 2023-12-21 154356](https://github.com/Keyfactor/ejbca-ce/assets/30307030/aff266bc-5bf1-4518-8615-4fa244b0264a)
After I finished up run build containers and check the logs, it give me the URL https://ejbca-node1:443/ejbca/adminweb/ to access the EJBCA CA UI. But it seems like it cannot be access, can I know why?

**Comment by primetomas** [comment]
That screenshot doesn't work. But I have a hunch. ejbca-node1 need to be a name that points to your docker host, typically localhost. Instead of adding ejbca-node1 to your /etc/hosts file you can just try https://localhost/ejbca/adminweb
